notes.txt

```diff
@@ -0,0 +1,117 @@
+## Module Resolution
```

Comment:
We should just include this into the readme as a Tech notes, or the README.md in SRC folder. So is accesible from github

Reply:
Think it's a good idea 👌 . Let's define in today's daily where we want to place this :) :)

Reply:
So the right place is going to be the same readme, let-s keep it simple : )

Reply:
👌 Great 😄 😄